advanced/Scripts/api.sh

```diff
@@ -51,6 +51,12 @@ TestAPIAvailability() {
             API_PORT=""
         else
             # API is available at this URL combination
+
+            if [ "${availabilityResonse}" = 200 ]; then
```

Comment:
/nitpick

```suggestion
            if [ "${availabilityResponse}" = 200 ]; then
```

Reply:
The variable is not my invention, it's been there already - including the typo. Adding this suggested change would make the code stop working

Reply:
Thanks for changing it, though :)